elements/campaign-display/campaign-display.test.js

```diff
@@ -14,6 +14,11 @@ describe('<campaign-display>', () => {
       campaignUrl,
       display: 'image',
     };
+
+    // Prevent setState warnings spamming the console
+    // We sould investigate if this is an issue with lib/bulbs-elements/store/store.js:60
+    CampaignDisplay.prototype.setState = chai.spy();
```

Comment:
Somehow we're calling setState before the component is fully mounted in DOM. (React demands that components be mounted before you `setState` on them.)

I think because this test is using shallowRenderer on `<CampaignDisplay/>` it's never mounting in DOM

Maybe store could check the mount status of components before calling setState?